config/clock.rb

```diff
@@ -50,4 +50,9 @@ class Clock
       TeacherTrainingPublicAPI::SyncAllProvidersAndCoursesWorker.perform_async
     end
   end
+
+  every(1.day, 'Generate export for Notifications', at: '23:59') do
```

Comment:
Although this is specified to run the same time as the TAD export, maybe we leave a few minutes between them for performance purposes?

Reply:
Shifted it to 23.57